### ISSUE TITLE
This project is useless

### ISSUE THREAD

**Issue by Zhomart** [kelseyhightower/nocode#1304]
I've tried doing nothing using this project. And it does completely nothing! Complete waste of my time.

**Comment by Khangeldy** [comment]
This project his best work. He only came to this world to show us this beautiful code. This code is priceless!

**Comment by nbarnes** [comment]
> I've tried doing nothing using this project. And it does completely nothing!

Working as intended?

**Comment by gaganbasra** [comment]
let me rephrase your issue "Working as expected" 👍
